src/main/java/me/champeau/gradle/japicmp/ignore/entity/Entity.java

```diff
@@ -0,0 +1,12 @@
+package me.champeau.gradle.japicmp.ignore.entity;
+
+//TODO: Implement version supporting
+public abstract class Entity {
+  private final String version;
```

Comment:
private final, no accessor?

Reply:
I not supported version checking yet, I will push new changes about it soon.

Reply:
Done